Update the makefile: Stop building it as a debug version by default,
remove the manual mangling of the include directories and don't build
the clockdiff, rdisc, tftp or rarpd components (everyone uses version
provided by other packages.)

--- new/Makefile	2005/11/14 13:34:06	1.1
+++ new/Makefile	2005/11/14 13:40:13
@@ -1,37 +1,17 @@
-# Path to parent kernel include files directory
-KERNEL_INCLUDE=/usr/src/linux/include
-LIBC_INCLUDE=/usr/include
-
-DEFINES= 
-
 #options if you have a bind>=4.9.4 libresolv (or, maybe, glibc)
 LDLIBS=-lresolv
-ADDLIB=
-
-ifeq ($(LIBC_INCLUDE)/socketbits.h,$(wildcard $(LIBC_INCLUDE)/socketbits.h))
-  ifeq ($(LIBC_INCLUDE)/net/if_packet.h,$(wildcard $(LIBC_INCLUDE)/net/if_packet.h))
-    GLIBCFIX=-Iinclude-glibc -include include-glibc/glibc-bugs.h
-  endif
-endif
-ifeq ($(LIBC_INCLUDE)/bits/socket.h,$(wildcard $(LIBC_INCLUDE)/bits/socket.h))
-  GLIBCFIX=-Iinclude-glibc -include include-glibc/glibc-bugs.h
-endif
-
-
-#options if you compile with libc5, and without a bind>=4.9.4 libresolv
-# NOT AVAILABLE. Please, use libresolv.
 
 CC=gcc
 # What a pity, all new gccs are buggy and -Werror does not work. Sigh.
 #CCOPT=-D_GNU_SOURCE -O2 -Wstrict-prototypes -Wall -g -Werror
-CCOPT=-D_GNU_SOURCE -O2 -Wstrict-prototypes -Wall -g
-CFLAGS=$(CCOPT) $(GLIBCFIX) -I$(KERNEL_INCLUDE) -I../include $(DEFINES) 
+CCOPT=-D_GNU_SOURCE -O2 -Wstrict-prototypes -Wall
+CFLAGS=$(CCOPT) -I$(KERNEL_INCLUDE) $(DEFINES) 
 
-IPV4_TARGETS=tracepath ping clockdiff rdisc arping tftpd rarpd
+IPV4_TARGETS=tracepath ping arping
 IPV6_TARGETS=tracepath6 traceroute6 ping6
 TARGETS=$(IPV4_TARGETS) $(IPV6_TARGETS)
 
-all: check-kernel $(TARGETS)
+all: $(TARGETS)
 
 
 tftpd: tftpd.o tftpsubs.o
